addons/l10n_br/models/template_br.py

```diff
@@ -35,3 +35,22 @@ def _get_br_res_company(self):
                 'account_purchase_tax_id': 'tax_template_in_icms_interno17',
             },
         }
+
+    @template('br', 'account.journal')
+    def _get_br_account_journal(self):
+        return {
+            'sale': {'l10n_br_invoice_serial': '1'},
```

Comment:
I'll wait for the decision on this field.

Reply:
We might need the field on the journal anyways.

Reply:
Yes, in the Journal we need it.

Reply:
@jco-odoo the main reason we set a default at all is so debugging/demoing is easier. I want to minimize the amount of setup needed for EDI. I think the default of 1 will always need to be changed because it's given by the government.

@vbe-odoo agree? JCO asks to set the serial to 1 on all journals instead of just on the initial ones created.

Reply:
Oh, I see... well, it won't be ideal to have on every new Journal that field = 1 by default, as it might change and not always be 1.
On the ones we create initially yes, but for the other new ones, I don't think so.

Reply:
@jorenvo talking with @jco-odoo let's keep as is -> only the 1st Journal created will have Serial = 1, and not new journals created